client/python/eidolon_ai_client/events.py

```diff
@@ -135,6 +135,7 @@ class AgentStateEvent(BaseStreamEvent):
     event_type: Literal["agent_state"] = "agent_state"
     state: str
     available_actions: List[str] = None  # this is filled in by the server, agents should leave the default
+    title: Optional[str] = None
```

Comment:
what does title represent here? just a human readable string for the state transition?

Reply:
oh, I see,  this is to represent the conversation title changing. I believe that should be a separate event type if we want to push that information to the ui. If we are worried about the events blowing out we can make a free form metadata event (that can likely encompasses agent state as well).